plugin/trino-hive/pom.xml

```diff
@@ -553,6 +553,7 @@
                                 <exclude>**/TestTrinoS3FileSystemAwsS3.java</exclude>
                                 <exclude>**/TestS3SelectQueries.java</exclude>
                                 <exclude>**/TestFullParquetReader.java</exclude>
+                                <exclude>**/TestParquetReader.java</exclude>
```

Comment:
natural expectation is that every excluded test is included somewhere
this one is not, so CI provides no guarantee that TestParquetReader works at all

maybe we should remove the test if we don't want to run it?
if we want to keep it for engineer's convenience, why not let the CI run it, so that it's guaranteed to work?

(at the very least, document why it's excluded without being included anywhere, and why it's worthwhile to do such an exceptional thing)

Reply:
Updated to move TestParquetReader into parquet test profile rather than dropping it entirely